system/admin/handlers.go

```diff
@@ -2200,6 +2200,15 @@ func deleteUploadHandler(res http.ResponseWriter, req *http.Request) {
 		return
 	}
 
+	// delete from file system, if good, we continue to delete
+	// from database, if bad error 500
+	err = deleteUploadFromDisk(t + ":" + id)
+	if err != nil {
+		log.Println(err)
+		res.WriteHeader(http.StatusInternalServerError)
+		return
+	}
+
 	err = db.DeleteUpload(t + ":" + id)
```

Comment:
nitpicky, but we could allocate the string for just one `(t + ":" + id)` and use in both delete func calls :)

Reply:
Yes, I saw that.  Minimal footprint ;)